pippy_app.py

```diff
@@ -404,6 +406,7 @@ def initialize_display(self):
         self.paths.append([_('My examples'), data_path])
 
         self._source_tabs = SourceNotebook(self, self._collab)
+        self._source_tabs._edit_toolbar = self._edit_toolbar
```

Comment:
If you intend to use `_edit_toolbar_` this way then it'll be better to make it public, it's different if you directly pass it as an arg to `SourceNotebook`.

Reply:
Oops, you are right I should have directly passed it to the constructor instead.